src/quo2/components/avatars/user_avatar/component_spec.cljs

```diff
@@ -0,0 +1,137 @@
+(ns quo2.components.avatars.user-avatar.component-spec
+  (:require [quo2.components.avatars.user-avatar.view :as user-avatar]
+            [test-helpers.component :as h]))
+
+(defonce mock-picture (js/require "../resources/images/mock2/user_picture_male4.png"))
+
+(defn- is-truthy
```

Comment:
why are these not in the test helpers file? 🤔

Reply:
What do you find confusing about `expect-to-be-truthy` vs `is-truthy`? Do you mean like the confusion is on the "assertion style", like behavior driven (expect bla bla), instead of the core Clojure way? I'm okay with both, but I prefer the shorter version with `is-`.

Reply:
@J-Son89  I'll move them rn.

I kept them as private helper functions because I was not sure if they were a good idea.

Agree with @ilmotta about using a shorter and Clojure style naming.

What should I do then?
After fixing this issue, I'll merge this PR

Reply:
sounds good 👍